ExtLibs/ProjNet/CoordinateSystems/Ellipsoid.cs

```diff
@@ -142,7 +142,7 @@ public static Ellipsoid Clarke1880
 			get
 			{
 				return new Ellipsoid(20926202, 0, 297, true, LinearUnit.ClarkesFoot, "Clarke 1880", "EPSG", 7034, "Clarke 1880", String.Empty,
-					"Clarke gave a and b and also 1/f=293.465 (to 3 decimal places).  1/f derived from a and b = 293.4663077�");
+					"Clarke gave a and b and also 1/f=293.465 (to 3 decimal places).  1/f derived from a and b = 293.4663077.");

```

Comment:
I think this change has nothing to do with the icon. So I think it should be another PR or remove this change from commit.

Reply:
I copied this part from github and is coping with paste. The character "..." seems to be garbled.
Clarke gave a and b and also 1 / f = 293.465 (to 3 decimal places). 1 / f derived from a and b = 293.4663077 ... "

Reply:
I see...
I'll below:
  1) close this PR.
  2) create new PR for fix frabled char.
  3) reopen this pr ( or create new PR same as  this. ).